site/en/docs/react_gemini_prompting.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>"View on Google AI" should now be "View on ai.google.dev"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google/generative-ai-docs/pull/255/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>